pkg/server/commands/list_objects_test.go

```diff
@@ -32,16 +34,20 @@ func TestNewListObjectsQuery(t *testing.T) {
 
 func TestListObjectsDispatchCount(t *testing.T) {
```

Comment:
This test is also flaky 😅 

```
~/GitHub/openfga/pkg/server/commands git:[double-queue-list-objects-dispatch-throttling]
go test -count=100 -race . -run="ListObjects"
--- FAIL: TestListObjectsDispatchCount (0.01s)
    reverse_expand.go:627: Unexpected call to *mocks.MockThrottler.Throttle([context.Background.WithValue(type storage.ctxKey, val <not Stringer>).WithValue(type typesystem.ctxKey, val <not Stringer>).WithValue(type typesystem.ctxKey, val <not Stringer>).WithDeadline(2024-05-08 11:24:39.512013 -0700 PDT m=+3.118123251 [2.998680958s]).WithCancel.WithValue(type trace.traceContextKeyType, val <not Stringer>).WithValue(type graph.ctxKey, val <not Stringer>).WithCancel.WithValue(type trace.traceContextKeyType, val <not Stringer>).WithValue(type trace.traceContextKeyType, val <not Stringer>).WithCancel]) at /Users/maria.inesparnisari/GitHub/openfga/pkg/server/commands/reverseexpand/reverse_expand.go:627 because: 
        expected call at /Users/maria.inesparnisari/GitHub/openfga/pkg/server/commands/list_objects_test.go:206 has already been called the max number of times
        expected call at /Users/maria.inesparnisari/GitHub/openfga/pkg/server/commands/list_objects_test.go:206 has already been called the max number of times
--- FAIL: TestListObjectsDispatchCount (0.01s)
    reverse_expand.go:627: Unexpected call to *mocks.MockThrottler.Throttle([context.Background.WithValue(type storage.ctxKey, val <not Stringer>).WithValue(type typesystem.ctxKey, val <not Stringer>).WithValue(type typesystem.ctxKey, val <not Stringer>).WithDeadline(2024-05-08 11:24:39.746553 -0700 PDT m=+3.352663835 [2.99973s]).WithCancel.WithValue(type trace.traceContextKeyType, val <not Stringer>).WithValue(type graph.ctxKey, val <not Stringer>).WithCancel.WithValue(type trace.traceContextKeyType, val <not Stringer>).WithValue(type trace.traceContextKeyType, val <not Stringer>).WithCancel]) at /Users/maria.inesparnisari/GitHub/openfga/pkg/server/commands/reverseexpand/reverse_expand.go:627 because: 
        expected call at /Users/maria.inesparnisari/GitHub/openfga/pkg/server/commands/list_objects_test.go:206 has already been called the max number of times
        expected call at /Users/maria.inesparnisari/GitHub/openfga/pkg/server/commands/list_objects_test.go:206 has already been called the max number of times
--- FAIL: TestListObjectsDispatchCount (0.01s)
    reverse_expand.go:627: Unexpected call to *mocks.MockThrottler.Throttle([context.Background.WithValue(type storage.ctxKey, val <not Stringer>).WithValue(type typesystem.ctxKey, val <not Stringer>).WithValue(type typesystem.ctxKey, val <not Stringer>).WithDeadline(2024-05-08 11:24:39.819467 -0700 PDT m=+3.425577126 [2.999727416s]).WithCancel.WithValue(type trace.traceContextKeyType, val <not Stringer>).WithValue(type graph.ctxKey, val <not Stringer>).WithCancel.WithValue(type trace.traceContextKeyType, val <not Stringer>).WithValue(type trace.traceContextKeyType, val <not Stringer>).WithCancel]) at /Users/maria.inesparnisari/GitHub/openfga/pkg/server/commands/reverseexpand/reverse_expand.go:627 because: 
        expected call at /Users/maria.inesparnisari/GitHub/openfga/pkg/server/commands/list_objects_test.go:206 has already been called the max number of times
        expected call at /Users/maria.inesparnisari/GitHub/openfga/pkg/server/commands/list_objects_test.go:206 has already been called the max number of times
FAIL
FAIL    github.com/openfga/openfga/pkg/server/commands  1.346s
FAIL
```